blocks/alternative_mutators/plus_minus/main.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview
+ */
+
+let plusMinus = {};
+
+function start() {
+  parseArguments();
+  workspace = Blockly.inject('blocklyDiv', options);
+  restoreSessionStorage();
+
+  /*workspace.addChangeListener(function(e) {
+    console.log(e);
+  })*/
+}
+
+document.addEventListener("DOMContentLoaded", function () { start() });
+
+var options = {};
+
+var workspace = null;
+
+
+// Playground specific functions.
+function parseArguments() {
```

Comment:
@samelhusseini to consider whether any of these functions (or similar) should go in the `dev-tools` package.